example_dags/example_databricks_workflow.py

```diff
@@ -59,7 +61,7 @@
         group_id=f"test_workflow_{USER}_{GROUP_ID}",
         databricks_conn_id=DATABRICKS_CONN_ID,
         job_clusters=job_cluster_spec,
-        notebook_params=[],
```

Comment:
Based on the 2.x spec, `notebook_params` are supposed to be dictionaries - we had previously assigned lists by mistake.
More on the spec can be found at: https://docs.databricks.com/dev-tools/api/latest/jobs.html#operation/JobsRunNow